README.md

```diff
@@ -26,16 +37,41 @@ p { font-size: 1em; }
 
 @media only screen and (max-width: 400px) {
   body { font-size: 130%; }  
-} 
-``` 
-
-In the code above, on line 1, 2, and 3 we are setting default styles. The body element's default text size is already 100%, but we have included it here to clarify the change that's made in the media query. On line 2 we set all `h1` elements to be 2.5em, and on line 3 we set the `p` element to 1em. So, the h1 is two and a half times larger than our paragraphs. In our media query on line 5 we set the query to trigger for device screens less than 400 pixels. When this occurs on line 6 we adjust the body font-size to 130%. This causes all elements within the body (h1, p) to inherit this change and grow 30% larger. Since we sized using em, you'll notice that the h1 and p adjust in proportion to each other even as they grow or shrink in size. This makes it very convenient to make size adjustments to one element, the body and thus affect all other type in our layout uniformly.
-
-### Column Count
-
-Another simple tip for easily adjusting type is when you might want to wrap text into separate columns. This is especially true on larger screen sizes where you have big bodies of text content. You can imagine that as a paragraph stretches across bigger screens the sentences start to get really long, and it becomes awkward for the reader to find the next line after they reach the end and have to bring their eyes back across the screen and drop to exactly the next line of text. Designers suggest that for optimum readability you want the measure of your text lines to be between 40 to 80 characters. Anything smaller or larger becomes a bit awkward to read. To tackle this we can make use of a lesser known, yet fairly well supported CSS `column-count` property.
+}
+```
 
-The default column-count is one, meaning text is contained within one column. When our screen gets bigger, it might be good to adjust the text into multiple columns; let's see an example:
+In the code above, on line 1, 2, and 3 we are setting default styles. The body
+element's default text size is already 100%, but we have included it here to
+clarify the change that's made in the media query. On line 2 we set all `h1`
+elements to be 2.5em, and on line 3 we set the `p` element to 1em. So, the h1 is
+two and a half times larger than our paragraphs. In our media query on line 5 we
+set the query to trigger for device screens less than 400 pixels. When this
+occurs on line 6 we adjust the body font-size to 130%. This causes all elements
+within the body (h1, p) to inherit this change and grow 30% larger.
+
+Since we sized using em, the h1 and p adjust in proportion to each other even as
```

Comment:
wrap uses of em  as in something you'd type in a property with ticks. When talking about the typographical idea, 'em' or 'Em' is fine.

also wrap HTML elements in ticks e.g. `h1` and `p`